styleguide/Components/ReactComponent/ReactComponent.js

```diff
@@ -31,9 +31,7 @@ const ReactComponent = ({ component, exampleMode }) => {
           ""
         )}`}
       >
-        <Caption Component="span" level="1" weight="regular">
-          Github
-        </Caption>
+        <Caption Component="span">Github</Caption>
```

Comment:
`span` у нас сейчас дефолтный `Component`, можно и его прибрать:

```suggestion
        <Caption>Github</Caption>
```